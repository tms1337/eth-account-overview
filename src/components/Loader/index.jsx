import React from "react";
import { createUseStyles } from "react-jss";

const Loader = () => {
  const styles = useStyles();

  return (
    <img
      className={styles.img}
      src="https://imgur.com/zZ1whKT.gif"
      alt="data is loading"
    />
  );
};

const useStyles = createUseStyles({
  img: {
    width: "250px",
    height: "25px"
  }
})

export default Loader;
