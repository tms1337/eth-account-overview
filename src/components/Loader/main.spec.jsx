import React from "react";
import renderer from "react-test-renderer";

import Loader from ".";

describe("Loader", () => {
  it("renders snapshot", () => {
    const component = renderer.create(<Loader />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
