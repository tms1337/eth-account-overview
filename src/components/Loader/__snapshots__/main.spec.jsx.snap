// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders snapshot 1`] = `
<img
  alt="data is loading"
  className="img-0-2-1"
  src="https://imgur.com/zZ1whKT.gif"
/>
`;
